backend/src/main/java/gov/cdc/usds/simplereport/db/model/PatientLink.java

```diff
@@ -4,13 +4,14 @@
 import java.util.Date;
 import javax.persistence.Column;
 import javax.persistence.Entity;
+import javax.persistence.FetchType;
 import javax.persistence.JoinColumn;
 import javax.persistence.OneToOne;
 
 @Entity
 public class PatientLink extends EternalAuditedEntity {
 
-  @OneToOne(optional = false)
+  @OneToOne(optional = false, fetch = FetchType.LAZY)
```

Comment:
I think we may actually want this to be eager in practice, but OTOH in practice N=1 for these.

Reply:
Removing this one.